### PR TITLE
feat: add trivy scans for separate images

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/trivy-scan.sh [hunk 1]
```diff
@@ -24,9 +24,12 @@ tar -xvzf "trivy_${TRIVY_VERSION}_${TRIVY_ARCH}.tar.gz"
 rm "trivy_${TRIVY_VERSION}_${TRIVY_ARCH}.tar.gz"
 chmod a+x trivy 
 
-./trivy --scanners vuln rootfs -f json --ignore-unfixed --severity HIGH,CRITICAL -o "${TRIVY_REPORT_JSON_PATH}" /
-./trivy --scanners vuln rootfs -f table --ignore-unfixed --severity HIGH,CRITICAL -o "${TRIVY_REPORT_TABLE_PATH}" /
+./trivy --scanners vuln rootfs -f json --skip-dirs /var/lib/containerd --ignore-unfixed --severity HIGH,CRITICAL -o "${TRIVY_REPORT_JSON_PATH}" /
 
+IMAGE_LIST=$(ctr -n k8s.io image list -q | grep -v sha256)
+for image in $IMAGE_LIST; do
+    ./trivy --scanners vuln image --ignore-unfixed --severity HIGH,CRITICAL -f table $image >> ${TRIVY_REPORT_TABLE_PATH}
```
Comment (alexeldeib) [inline vhdbuilder/packer/trivy-scan.sh:31]: I wonder if we should do json. I think table is good for now, but we'll potentially want json arrays for automation

File: vhdbuilder/packer/trivy-scan.sh [hunk 2]
```diff
@@ -24,9 +24,12 @@ tar -xvzf "trivy_${TRIVY_VERSION}_${TRIVY_ARCH}.tar.gz"
...
+done
```
Comment (alexeldeib) [inline vhdbuilder/packer/trivy-scan.sh:33]: can you also remove the trivy DB image? I think trivy has a `--reset` or something, or you can manually remove the DB container image.

**Review by alexeldeib — APPROVED** [review]
comments are okay for follow up.

File: vhdbuilder/packer/trivy-scan.sh (hunk 2 above)
Comment (alexeldeib) [reply vhdbuilder/packer/trivy-scan.sh:33]: I suppose we should really do this one...docker hub image I think

Comment (ganeshkumar5699) [reply vhdbuilder/packer/trivy-scan.sh:37]: Sure, will look into this

File: vhdbuilder/packer/trivy-scan.sh (hunk 1 above)
Comment (ganeshkumar5699) [reply vhdbuilder/packer/trivy-scan.sh]: Yes, can add json output too

**Pull request merged**